src/patternfly/components/ClipboardCopy/clipboard-copy.scss

```
@@ -18,8 +18,8 @@
   --#{$clipboard-copy}__expandable-content--BorderInlineEndWidth: var(--pf-t--global--border--width--control--default);
   --#{$clipboard-copy}__expandable-content--BorderBlockEndWidth: var(--pf-t--global--border--width--control--default);
   --#{$clipboard-copy}__expandable-content--BorderInlineStartWidth: var(--pf-t--global--border--width--control--default);
-  --#{$clipboard-copy}__expandable-content--BorderColor: var(--pf-t--global--border--color--default);
-  --#{$clipboard-copy}__expandable-content--BorderRadius: var(--pf-t--global--border--radius--small);
+  --#{$clipboard-copy}__expandable-content--BorderColor: var(--pf-t--global--border--color--control--default);
+  --#{$clipboard-copy}__expandable-content--BorderRadius: var(--pf-t--global--border--radius--form-element);
```

Stylelint (17.2.0)

[error] Expected custom property name "--pf-t--global--border--radius--form-element" to be kebab-case (custom-property-pattern)

(custom-property-pattern)